Swift Compiler Errors

ERROR HERE:
flutter/.pub-cache/hosted/pub.dartlang.org/screenshot_callback-1.1.5/ios/Classes/SwiftScreenshotCallbackPlugin.swift:27:42: Type 'NSNotification.Name' has no member 'UIApplication'

AND:
.pub-cache/hosted/pub.dartlang.org/screenshot_callback-1.1.5/ios/Classes/SwiftScreenshotCallbackPlugin.swift:27:40: error:
    'UIApplicationUserDidTakeScreenshot' has been renamed to 'UIApplication.userDidTakeScreenshotNotification'
              forName: NSNotification.Name.UIApplicationUserDidTakeScreenshot,
                                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                           UIApplication.userDidTakeScreenshotNotification
    UIKit.NSNotification.Name:43:23: note: 'UIApplicationUserDidTakeScreenshot' was obsoleted in Swift 4.2
        public static let UIApplicationUserDidTakeScreenshot: NSNotification.Name

--- a/ios/Classes/SwiftScreenshotCallbackPlugin.swift
+++ b/ios/Classes/SwiftScreenshotCallbackPlugin.swift
@@ -24,7 +24,7 @@ public class SwiftScreenshotCallbackPlugin: NSObject, FlutterPlugin {
             SwiftScreenshotCallbackPlugin.observer = nil;
         }
         SwiftScreenshotCallbackPlugin.observer = NotificationCenter.default.addObserver(
-          forName: NSNotification.Name.UIApplicationUserDidTakeScreenshot,
+          forName: UIApplication.userDidTakeScreenshotNotification,
           object: nil,
           queue: .main) { notification in
           if let channel = SwiftScreenshotCallbackPlugin.channel {

